ts/features/payments/checkout/hooks/usePagoPaPayment.ts

```diff
@@ -39,7 +38,6 @@ type UsePagoPaPayment = {
   ) => void;
   // This is a temporary flag to tell that the new payment flow is enabled and can be used
```

Comment:
I think these comments can be removed without too!

Reply:
yess good catch! Thanks!